main.py

```diff
@@ -163,7 +163,11 @@ def actuators(self, message, nick, pm):
             elif "!meow" in message_lower:
                 result = random_cat.random_cat_pic()
                 self.send_msg(result, nick, pm)
-            
+
+            elif "!catfact" in message_lower:
+		result = random_cat_fact.random_cat_facts()
```

Comment:
TabError: inconsistent use of tabs and spaces in indentation
```suggestion
                result = random_cat_fact.random_cat_facts()
                self.send_msg(result, nick, pm)                
```